src/libraries/System.Formats.Asn1/ref/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
```

Comment:
This seems like it was copied from somewhere but it is not used since your project doesn't target netfx so you can remove